Src/IronPython/Runtime/MemoryView.cs

```diff
@@ -543,6 +552,69 @@ public sealed class MemoryView : ICodeFormattable, IWeakReferenceable {
             }
         }
 
+        public object this[[NotNull]PythonTuple index] {
+            get {
+                CheckBuffer();
+
+                int[] tupleValues = FixTuple(index);
```

Comment:
From a perf point of view would be nice to avoid the array allocation on every item get/set. Maybe `FixTuple` could be changed to `GetFlatIndex` or something like that?